tools/correlation/cor.xml

```diff
@@ -1,33 +1,48 @@
-<tool id="cor2" name="Correlation" version="1.0.0">
-  <description>for numeric columns</description>
-  <requirements>
-    <requirement type="package" version="1.0.3">rpy</requirement>
-  </requirements>
-  <command interpreter="python">cor.py $input1 $out_file1 $numeric_columns $method</command>
-  <inputs>
-    <param format="tabular" name="input1" type="data" label="Dataset" help="Dataset missing? See TIP below"/>
-    <param name="numeric_columns" label="Numerical columns" type="data_column" numerical="True" multiple="True" data_ref="input1" help="Multi-select list - hold the appropriate key while clicking to select multiple columns" />
-    <param name="method" type="select" label="Method">
-      <option value="pearson">Pearson</option>
-      <option value="kendall">Kendall rank</option>
-      <option value="spearman">Spearman rank</option>
-    </param>
-  </inputs>
-  <outputs>
-    <data format="txt" name="out_file1" />
-  </outputs>
-  <tests>
-    <!--
-    Test a tabular input with the first line being a comment without a # character to start
-    -->
-    <test>
-      <param name="input1" value="cor.tabular" />
-      <param name="numeric_columns" value="2,3" />
-      <param name="method" value="pearson" />
-      <output name="out_file1" file="cor_out.txt" />
-    </test>
-  </tests>
-  <help>
+<tool id="cor2" name="Correlation" version="1.0.1">
+    <description>for numeric columns</description>
+    <requirements>
+        <requirement type="package" version="2.9.4">rpy2</requirement>
+    </requirements>
+    <command>
+      python '$__tool_directory__/cor.py'
+        '$input1'
+        '$out_file1'
+        $numeric_columns
+        $method
+    </command>
+    <inputs>
+        <param format="tabular" name="input1" type="data" label="Dataset" help="Dataset missing? See TIP below"/>
+        <param name="numeric_columns" label="Numerical columns" type="text" multiple="True"
+            data_ref="input1" help="Multi-select list - hold the appropriate key while clicking to select multiple columns" />
+        <!--param name="numeric_columns" label="Numerical columns" type="data_column" numerical="True" multiple="True"

```

Comment:
@guerler do you remember that we talk once about the data_column parameter and that it would be nice to allow an `overwrite mode` like in the workflow form. So a user could choose to insert numbers instead of choosing the fields. 
I get this request now a lot in communities that do more statistics on large files (>30 columns).

This PR demonstrates a simple tool that can take `2-3` as an indicator of the columns instead of `2,3`. Tests are passing, maybe you can use this to explore some UI options? The idea is to offer the user some more advances selections like `2,5,10-30,40` in an advanced mode next to the traditional `data_column` view.